modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {KialiProduct}
+
+{KialiProduct} provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.
+
+Kiali provides observability into the Service Mesh running on OpenShift Container Platform. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your namespace in real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
+
+Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating {TempoName} and {OTELName} into the Kiali console.
+
+== {KialiProduct} architecture
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali is composed of two components: the Kiali application and the Kiali console.
```

Comment:
We normally call it the Kiali Server (not the Kiali application). And, in fact, on the very next line (line 20), it does use the term `Kiali Server`. For consistency here in the docs, and how we historically call it, I would change application to server:
```suggestion
Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali is composed of two components: the Kiali Server and the Kiali console.
```